plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.tsx

```diff
@@ -89,57 +89,64 @@ export default function MySubscriptions(): JSX.Element {
 
 	return (
 		<div className="woocommerce-marketplace__my-subscriptions">
-			<section className="woocommerce-marketplace__my-subscriptions__installed">
-				<header className="woocommerce-marketplace__my-subscriptions__header">
-					<div>
-						<h2 className="woocommerce-marketplace__my-subscriptions__heading">
-							{ __( 'Installed on this store', 'woocommerce' ) }
-						</h2>
-						<span className="woocommerce-marketplace__my-subscriptions__table-description">
-							{ installedTableDescription }
-						</span>
-					</div>
-					<div>
-						<Button
-							href={ updateConnectionUrl }
-							className="woocommerce-marketplace__refresh-subscriptions"
-						>
-							<img
-								src={ RefreshIcon }
-								alt={ __(
-									'Refresh subscriptions',
+			{ subscriptionsInstalled.length > 0 && (
+				<section className="woocommerce-marketplace__my-subscriptions__installed">
+					<header className="woocommerce-marketplace__my-subscriptions__header">
+						<div>
+							<h2 className="woocommerce-marketplace__my-subscriptions__heading">
+								{ __(
+									'Installed on this store',
 									'woocommerce'
 								) }
-							/>
-							{ __( 'Refresh', 'woocommerce' ) }
-						</Button>
-					</div>
-				</header>
-				<InstalledSubscriptionsTable
-					isLoading={ isLoading }
-					rows={ subscriptionsInstalled.map( ( item ) => {
-						return installedSubscriptionRow( item );
-					} ) }
-				/>
-			</section>
+							</h2>
+							<span className="woocommerce-marketplace__my-subscriptions__table-description">
+								{ installedTableDescription }
+							</span>
+						</div>
+						<div>
+							<Button
+								href={ updateConnectionUrl }
+								className="woocommerce-marketplace__refresh-subscriptions"
+							>
+								<img
```

Comment:
Since this is now behind a if statement, I don't see the **Refresh** button when I don't have and plugins installed. 

![image](https://github.com/woocommerce/woocommerce/assets/10389957/564ba6d6-348e-4262-8bed-8b1f4548cbcc)

Reply:
It will show the header now. The design also has a section for this case, but I'm not sure if someone already implemented it.